pkg/process/util/network_filter.go

```diff
@@ -11,14 +11,19 @@ import (
 // ConnectionFilter holds a user-defined blacklisted IP/CIDR, and ports
 type ConnectionFilter struct {
 	IP       *net.IPNet
-	Ports    map[uint16]struct{}
-	AllPorts bool
+	Ports    map[uint16]ConnType
+	AllPorts ConnType
+}
+
+type ConnType struct {
```

Comment:
How about calling this  `ConnTypeFilter` or something like that. `ConnType` is very close to ConnectionType which is used by the agent model

Reply:
Same as above. I think it could make sense to bring this to `ebpf` package and use `ebpf.ConnectionType`

Reply:
I don't think the types are the same:

ebpf.ConnectionType is either TCP or UDP. `ConnTypeFilter` is one of `tcp`, `udp`, or `tcp` && `udp`